EPA-MVS/mvs_config.json

```diff
@@ -0,0 +1,570 @@
+            },
+            "specific_costs_om": {
+                "unit": "currency/year",
+                "value": 20.0
+            },
+            "unit": "kVA",
+            "energyVector": "Heat",
+            "inflow_direction": "Electricity",
+            "outflow_direction": "Heat",
+            "installedCap": {
+                "unit": "unit",
+                "value": 250.0
+            },
+            "optimizeCap": {
+                "unit": "bool",
+                "value": false
+            }
+        }
+    },
+    "energyProduction": {
+    },
+    "energyStorage": {},
+    "energyBusses": {
+        "Electricity": {
+            "label": "Electricity",
+            "energyVector": "Electricity"
+        },
+        "Heat": {
+            "label": "Heat",
+            "energyVector": "Heat"
+        }
+    },
+    "energyProviders": {
+        "Grid_DSO": {
+            "label": "Grid_DSO",
+            "unique_id": "afd268c6-0c8d-4434-ac4d-92b106a098c9",
+            "type_oemof": "source",
+            "energy_price": {
+                "unit": "currency/kWh",
+                "value": 0.1
+            },
+            "feedin_tariff": {
+                "unit": "currency/kWh",
+                "value": 0.0
+            },
+            "peak_demand_pricing": {
+                "unit": "currency/kW",
+                "value": 0.0
+            },
+            "peak_demand_pricing_period": {
+                "unit": "times per year (1,2,3,4,6,12)",
+                "value": 1
+            },
+            "renewable_share": {
+                "unit": "factor",
+                "value": 0.1
+            },
+            "energyVector": "Electricity",
+            "outflow_direction": "Electricity",
+            "optimizeCap": {
+                "unit": "bool",
+                "value": true
+            },
+            "unit": "kWh",
+            "inflow_direction": "Electricity",
+            "emission_factor": {
+                "value": 0
+            }
+        },
+        "Heat_DSO": {
+            "label": "Heat_DSO",
+            "unique_id": "185bd7bc-f104-473a-8bd0-6cc52a53a6c0",
+            "type_oemof": "source",
+            "energy_price": {
+                "unit": "currency/kWh",
+                "value": 0.056
+            },
+            "feedin_tariff": {
+                "unit": "currency/kWh",
+                "value": 0.0
+            },
+            "peak_demand_pricing": {
+                "unit": "currency/kW",
+                "value": 0.0
+            },
+            "peak_demand_pricing_period": {
+                "unit": "times per year (1,2,3,4,6,12)",
+                "value": 1
+            },
+            "renewable_share": {
+                "unit": "factor",
+                "value": 0.1
+            },
+            "energyVector": "Heat",
+            "outflow_direction": "Heat",
+            "optimizeCap": {
+                "unit": "bool",
+                "value": true
```

Comment:
This should be false

Reply:
Weird, it actually does not have to be anything, and if I open my `MVS_outputs` it is also set to `True`. Did not work for you?